CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 1.0.0-beta.0
```

Comment:
what about a fancy name to the first beta???? 😃

Reply:
Lol I'm on it

Reply:
That was harder than I expected